frontend/model/state.js

```diff
@@ -145,9 +145,10 @@ const getters = {
   },
   // list of group names and contractIDs
   groupsByName (state) {
-    return Object.entries(store.state.contracts)
-      .filter(([key, value]) => value.type === 'GroupContract')
-      .map(([key]) => ({ groupName: state[key].groupName, contractID: key }))
+    const { contracts } = store.state.contracts
+    return Object.keys(contracts)
+      .filter((key) => contracts[key].type === 'GroupContract')
+      .map((key) => ({ groupName: state[key].groupName, contractID: key }))
```

Comment:
Changed it to the Object.keys() version only for the sake of solving flow error even though using Object.entries() is obviously simpler and cleaner. 
please refer to:
- https://github.com/facebook/flow/issues/5838,
- https://github.com/facebook/flow/blob/master/lib/core.js#L48